.env.example

```diff
@@ -0,0 +1,16 @@
+# Account that is going to pay for the demo
```

Comment:
I would add .env.example.json file and avoid  "dotenv": "^16.0.0" extra dependency. 
`{
    "OPERATOR_ID": "0.0.12710106",

    "OPERATOR_KEY": "302e020100300506032b657004220420bc45334a1313725653d3513fcc67edb15f76985f537ca567e2177b0be9906d49",

    "DID_IDENTIFIER": "did:hedera:testnet:z6Mkqxn1SZyuPkgbuPNaejANNwZvBscQL1zykDoPfu5oS8G7_0.0.30767279",

    "DID_PRIVATE_KEY": "302e020100300506032b65700422042055aaabadc8ae33bf9737114267fc60efddaade7fc827749b4292b75a21c48f38"
}`

Reply:
Thanks @vijayshiyani , good point 👍